shared/settings/about-container.tsx

```diff
@@ -28,6 +27,7 @@ const mapDispatchToProps = (dispatch, {navigateUp, navigateAppend}) => ({
 })
 
 const connectedHeaderHoc = Container.compose(
+  // @ts-ignore codemod-issue
   Container.connect<OwnProps, _, _, _, _>(
```

Comment:
This should be fine if you remove the `<...>` i.e. it becomes `Container.connect(`